config/gitleaks.toml

```diff
@@ -30,7 +30,7 @@ title = "gitleaks config"
 	tags = ["email"]
 	[rules.allowlist]
 		regexes = ['''(?i)@(github.com|test.org)''',
-			'''(?i)@(trio.dhs.gov|cisa.dhs.gov|beta.dhs.gov|hq.dhs.gov)''',
+			'''(?i)@(beta|cisa|gwe.cisa|hq|trio).dhs.gov''',
```

Comment:
Do we need to escape periods here so they don't match any ol' character?

Reply:
Since we weren't doing it before, I'm hesitant to change it now because I don't have time to test it out.

Reply:
See #92.